numpy/core/einsumfunc.py

```diff
@@ -11,7 +11,7 @@
 
 __all__ = ['einsum', 'einsum_path']
 
-einsum_symbols = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+einsum_symbols = string.ascii_letters
```

Comment:
I could discard `einsum_symbols` as well, and use `string.ascii_letters` everyhwere.